### DISCUSSION TITLE
Using this dependency in a node 18 project

### DISCUSSION THREAD

**Discussion by montycheese** [MasterKale/SimpleWebAuthn#590] [General]
Hi there, I am investigating using this package inside a node 18 project. I see that version 10 is node 20 only. Not having any context on version 9, are there any hesitations on using version 9? (Aside from not getting any bug fixes that are introduced in version 10 and beyond)

**Comment by MasterKale** [comment]
There weren't any major internal changes to the library with v10, it was primarily a method API update. I can't see any reason to stick with v9 if you can't upgrade off of Node 18, so long as nothing in the release notes is alarming to you in anyway https://github.com/MasterKale/SimpleWebAuthn/releases/tag/v10.0.0
